front_end/entrypoints/inspector_main/CohtmlPanel.ts

```diff
@@ -437,34 +521,39 @@ export class CohtmlPanelView extends UI.Widget.VBox implements SDK.TargetManager
       option.textContent = options[index];
     }
     return select as HTMLSelectElement;
-}
+  }
 
-  private createButton(label: string, desc: string, parent: HTMLElement , click : () => void) {
+  private createButton(label: string, desc: string, parent: HTMLElement, click: () => void, subDesc?: string) {
     const newButtonBlock = parent.createChild('div');
     newButtonBlock.classList.add('button-block');
 
-    const buttonItem = newButtonBlock.createChild('span');
-    buttonItem.classList.add('button-link');
-    buttonItem.innerText = label;
-    buttonItem.onclick = click;
+    const buttonItem = UI.UIUtils.createTextButton(i18nString(label), click, 'cohtml-button');
+    UI.ARIAUtils.markAsLink(buttonItem);
 
-    const descItem = newButtonBlock.createChild('span');
+    const descItem = newButtonBlock.createChild('div');
     descItem.classList.add('button-desc');
     descItem.innerText = desc;
+    newButtonBlock.appendChild(descItem);
+
+    if (subDesc) {
```

Comment:
```suggestion
    if (subDescription) {
```
? for consistency :D

Reply:
Fixed